cpp/models/abm/location.cpp

```diff
@@ -197,5 +203,17 @@ Eigen::Ref<const Eigen::VectorXi> Location::get_subpopulations() const
     return Eigen::Map<const Eigen::VectorXi>(m_subpopulations.data(), m_subpopulations.size());
 }
 
+/*
+For every location we have a transmission factor that is nomalized to m_capacity.volume / m_capacity.persons of 
+the location "Home", which is 66. We multiply this rate with the factor m_capacity.persons / m_capacity.volume of the 
+considered location. For the dependency of the transmission risk on the occupancy we use a linear approximation which 
+leads to: 66 * (m_capacity.persons / m_capacity.volume) * (m_num_persons / m_capacity.persons).
+*/
+double Location::compute_relative_transmission_risk()
+{
+    return m_capacity.volume != 0 && m_capacity_adapted_transmission_risk ? 66.0 * m_num_persons / m_capacity.volume
```

Comment:
1. add a (...) so that we know directly that m_capacity.volume != 0 && m_capacity_adapted_transmission_risk is one statement.

Reply:
good remark. 

It would always opt for more parentheses than for less if it makes && or || signs directly clear